app/models/request_book.rb

```diff
@@ -9,11 +9,18 @@ class RequestBook < ApplicationRecord
                    greater_than_or_equal_to: Settings.request.amount.min}
 
   validate :amount_validation
+  validate :book_validation
```

Comment:
```suggestion
  validate :book_validation, if: -> {book.present? && amount.present?}
```
Viết vậy người khác đọc dễ hiểu hơn, với viết như vậy thì bỏ line 23 đi nhé